crates/holochain_state/src/query.rs

```diff
@@ -338,6 +341,37 @@ impl<'stmt> Store for Txn<'stmt, '_> {
         }
     }
 
+    fn get_warrants_for_basis(&self, hash: &AnyLinkableHash) -> StateQueryResult<Vec<Warrant>> {
+        let warrants = self
+            .txn
+            .prepare_cached(
+                "
+            SELECT
+            Action.blob
+            FROM Action
+            WHERE base_hash = :hash
+            AND type = :type
+            ",
```

Comment:
Inline sql always makes me sad... but I guess it's consistent with the current codebase.